packages/make-styles/src/runtime/resolveStyleRules.test.ts

```diff
@@ -583,6 +583,15 @@ describe('resolveStyleRules', () => {
     });
   });
 
+  describe('output', () => {
+    it('output for properties that do not depend on text direction contains less members', () => {
```

Comment:
nit: improve `describe` `it` output format

```suggestion
    it('contains less members for properties that do not depend on text direction', () => {
```